File: Java/프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        String s = String.valueOf(age);
        
        for(int i=0; i<s.length(); i++) {
            char num = s.charAt(i);
            char nn = (char)('a' +( num - '0'));
                answer += nn;
        }
        
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.js
function solution(age) {
    var answer = '';
    a = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j'];
    age = age.toString();
    
    for(let i=0; i<age.length; i++){
        answer += a[age[i]];
    }
    return answer;
}